pkg/protocol/engine/ledger/tests/output.go

```diff
@@ -9,7 +9,7 @@ func (m *MockedOutput) Size() int {
 	panic("implement me")
 }
 
-func (m *MockedOutput) VBytes(_ *iotago.RentStructure, _ iotago.VBytesFunc) uint64 {
+func (m *MockedOutput) VBytes(_ *iotago.RentStructure, _ iotago.VBytesFunc) iotago.VBytes {
```

Comment:
You need to update the iota.go dependency in the go.mod. The CI is not using a go.work file